tests/ui/closures/missing-body.rs

```diff
@@ -0,0 +1,8 @@
+// Checks that the compiler complains about the missing closure body and does not
+// crash.
+// This is a regression test for <https://github.com/rust-lang/rust/issues/143128>.
+
+fn main() { | b : [str; _] | }
```

Comment:
```suggestion
fn main() { |b: [str; _]| {}; }
```

error minimization without avoiding the ICE

Reply:
Shorter, nice!